packages/core/src/styles/utils/_legacy-spinner.scss

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+// TODO: progress spinner CSS deprecated, will be replaced by cds-progress
+
+$clr-baseline-int: 24 !default;
+$clr-baseline-denominator: 20 !default;
+
+@function convertBaselineToBase20($size: 24) {
+  $clr-baseline-to-basefontsize-multiplier: $clr-baseline-int/$clr-baseline-denominator;
+  @return $size/$clr-baseline-int * $clr-baseline-to-basefontsize-multiplier * 1rem;
+}
+
+$clr-color-hue: 198;
+$clr-color-neutral-saturation: 0%;
+$clr-color-neutral-0: hsl($clr-color-hue, $clr-color-neutral-saturation, 100%);
+$clr-color-action-600: hsl($clr-color-hue, 100%, 32%);
+$clr-color-neutral-1000: hsl($clr-color-hue, $clr-color-neutral-saturation, 0%);
+$clr_baselineRem_0_125: convertBaselineToBase20(3);
+$clr_baselineRem_3: convertBaselineToBase20(72);
+$clr_baselineRem_1_5: convertBaselineToBase20(36);
+$clr_baselineRem_0_75: convertBaselineToBase20(18);
+$clr_baselineRem_0_5416: convertBaselineToBase20(13);
+
+@import './../mixins/mixins';
+@import './../../../../angular/projects/clr-angular/src/progress/spinner/variables.spinner';
+@import './../../../../angular/projects/clr-angular/src/image/icons.clarity';
+@import './../../../../angular/projects/clr-angular/src/progress/spinner/spinner.clarity';
```

Comment:
Combined all the mixins into one file and removed all the legacy `$clr-` vars. These are the only ones left which are needed for the circular progress to be replaced with `cds-progress`